packages/fxa-settings/src/components/Nav/index.tsx

```diff
@@ -21,7 +21,7 @@ export const Nav = () => {
       primaryEmail
     )}`;
 
-  const activeClasses = 'font-bold text-blue-500 rounded-sm';
+  const activeClasses = 'font-bold text-blue-500 rounded-sm bg-grey-50';
```

Comment:
@karinabk I know you sent me a screenshot of this on Matrix and I _swear_ it looked a tad darker to me than this😅

<img width="280" alt="image" src="https://user-images.githubusercontent.com/13018240/113353022-b3c23900-9302-11eb-8b09-4caccf87abec.png">

I searched the codebase for `bg-grey-50`, and the only other place we seem to be using it is our [internal design guide](https://storage.googleapis.com/mozilla-storybooks-fxa/commits/28ce48d4925fdab4b8ad0d46afedd47893455f1c/fxa-settings/index.html?path=/story/design-guide-settings--colors). Because of this, and because `bg-grey-10` and `bg-grey-50` are almost visually identically similar, I think we go ahead and should darken `50` to be a better in-between color for `10` and `100`.

Will you update the `tailwind.js` file to reflect the BG color shown in the image from the original issue, `#F0F0F4`? 🙂 You can find it under "colors" in that file. Also, I think we might want to leave the padding at `py-1`... The `1` class outputs `.25rem` padding top/bottom, which matches the design at `4px` of top/bottom padding (our default "rem" size is `16px`). Thanks so much for putting a PR in for this!

Reply:
I updated the PR. Thanks for your help! :blush: